test/e2e/basics/render.test.js

```diff
@@ -93,6 +95,43 @@ test.describe("Render MapML resources test", () => {
             (viewer) => viewer.getAttribute('projection'));
         expect(projection).toEqual("OSMTILE");
     }, {times: 1});
+    
+    test("Do not render map from application/xml document when 'Render' is not checked", async () => {
```

Comment:
This test does not seem to fail when the implementation is removed, will take a look at this tomorrow.